spec/factories/orphan_filters.rb

```diff
@@ -23,8 +23,8 @@ class OrphanFilter < Hash
         province_code: orphan.province_code,
         original_address_city: orphan.original_address.city,
         priority: orphan.priority,
-        sponsorship_status: orphan.sponsorship_status,
-        status: orphan.status,
+        sponsorship_status: Orphan.sponsorship_statuses[orphan.sponsorship_status], #digit
+        status: Orphan.statuses[orphan.status], #digit
```

Comment:
this should return an intiger, not the string.
Factories should return same attributes as db records do.